Core/src/org/sleuthkit/autopsy/discovery/search/DiscoveryAttributes.java

```diff
@@ -693,6 +732,8 @@ public void addAttributeToResultFiles(List<Result> results, SleuthkitCase caseDb
         INTERESTING_ITEM_SET(new InterestingItemAttribute(), Bundle.DiscoveryAttributes_GroupingAttributeType_interestingItem_displayName()),
         FILE_TAG(new FileTagAttribute(), Bundle.DiscoveryAttributes_GroupingAttributeType_tag_displayName()),
         OBJECT_DETECTED(new ObjectDetectedAttribute(), Bundle.DiscoveryAttributes_GroupingAttributeType_object_displayName()),
+        MOST_RECENT_DATE(new MostRecentActivityDateAttribute(), Bundle.DiscoveryAttributes_GroupingAttributeType_mostRecentDate_displayName()),
+        FIRST_DATE(new MostRecentActivityDateAttribute(), Bundle.DiscoveryAttributes_GroupingAttributeType_firstDate_displayName()),
```

Comment:
This looks like a cut and paste error, I think it should be `FirstActivityDateAttribute()`

Reply:
yes once again